.circleci/config.yml

```diff
@@ -45,7 +45,9 @@ store_bazel_profile: &store_bazel_profile
 set_bazelrc: &set_bazelrc
   run:
     name: Set bazelrc
-    command: cat .circleci/bazelrc >> .bazelrc
+    command: |
+      cat .circleci/bazelrc >> .bazelrc
+      echo .bazelrc >> .gitignore
```

Comment:
If this work, add a comment about stamping

Reply:
Maybe put a full bazelrc file in `.circleci` dir and use that for CI(as Brian mentioned) so we don't have to modify the root one every time we run the CI job?